crates/ln-dlc-node/src/node/mod.rs

```diff
@@ -180,6 +182,14 @@ pub struct Node<S: TenTenOneStorage, N: Storage> {
     pub probes: Probes,
 }
 
+/// An on-chain network fee for a transaction
+pub enum Fee {
+    /// A fee given by the transaction's priority
+    Priority(ConfirmationTarget),
+    /// A fee given by an absolute amount
```

Comment:
I assume the absolute amount is in sats/vbyte? If so, can you update the doc please?